bin/wasabi.sh

```diff
@@ -255,9 +255,9 @@ package() {
   version=$(fromPom . build project.version)
   # FIXME: server ip
   server="http://localhost:8080"
-  home=$(fromPom . build application.home)
+  home=$(fromPom ./modules/main build application.home)
```

Comment:
Oh, that must have slipped through while we removed the python dependencies, great catch!